### PR TITLE
[Oracle] README updates regarding mandatory user permission

### PR DESCRIPTION
**Pull request by agithomas**
- Enhancement

## Proposed commit message

README updates regarding mandatory user permission.
This helps to assign permissions to Oracle user for configuring Oracle Integration

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] elastic-package stack up -v -d --services package-registry
- [x] Manual content verification

## How to test this PR locally

elastic-package build
elastic-package stack up -v -d --services package-registry

## Related issues

- Closes https://github.com/elastic/ingest-docs/issues/226

## Screenshots

image

### CODE REVIEW

File: packages/oracle/_dev/build/docs/README.md [hunk 1]
```diff
@@ -96,13 +96,23 @@ The `database_audit` dataset collects Oracle Audit logs.
 
 Tablespace metrics describes the tablespace usage metrics of all types of tablespaces in the oracle database.
 
+To collect the Tablespace metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:99]:
```suggestion
To collect the Tablespace metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```
For accessibility purposes, terms like "below" or "above" that indicate directions should be avoided.
Check also [How to create accessible web content](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit#heading=h.iykpo698d18b).

File: packages/oracle/_dev/build/docs/README.md [hunk 2]
```diff
@@ -96,13 +96,23 @@ The `database_audit` dataset collects Oracle Audit logs.
...
+- `DBA_FREE_SPACE`
+
 {{fields "tablespace"}}
 
 {{event "tablespace"}}
 
 ### Sysmetrics 
 
-The system metrics value captured for the most current time interval for the long duration (60-seconds) are mentioned below
+The system metrics value captured for the most current time interval for the long duration (60-seconds) are mentioned below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:111]:
```suggestion
The system metrics value captured for the most current time interval for the long duration (60-seconds) are listed in the following table. 
```

File: packages/oracle/_dev/build/docs/README.md [hunk 3]
```diff
@@ -96,13 +96,23 @@ The `database_audit` dataset collects Oracle Audit logs.
...
 {{fields "tablespace"}}
 
 {{event "tablespace"}}
 
 ### Sysmetrics 
 
-The system metrics value captured for the most current time interval for the long duration (60-seconds) are mentioned below
+The system metrics value captured for the most current time interval for the long duration (60-seconds) are mentioned below. 
+
+To collect the Sysmetrics metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the view mentioned below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:113]:
```suggestion
To collect the Sysmetrics metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following view:
```

File: packages/oracle/_dev/build/docs/README.md [hunk 4]
```diff
@@ -112,6 +122,11 @@ The system metrics value captured for the most current time interval for the lon
 
 A Program Global Area (PGA) is a memory region that contains data and control information for a server process. It is nonshared memory created by Oracle Database when a server process is started. Access to the PGA is exclusive to the server process. Metrics concerning Program Global Area (PGA) memory are mentioned below.
 
+To collect the Memory metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:125]:
```suggestion
To collect the Memory metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```

File: packages/oracle/_dev/build/docs/README.md [hunk 5]
```diff
@@ -120,6 +135,10 @@ A Program Global Area (PGA) is a memory region that contains data and control in
 
 The System Global Area (SGA) is a group of shared memory structures that contain data and control information for one Oracle Database instance. Metrics concerning System Global Area (SGA) memory are mentioned below.
 
+To collect the System Statistics metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the view mentioned below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:138]:
```suggestion
To collect the System Statistics metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following view:
```

File: packages/oracle/_dev/build/docs/README.md [hunk 6]
```diff
@@ -128,6 +147,16 @@ The System Global Area (SGA) is a group of shared memory structures that contain
 
 Performance metrics give an overview of where time is spent in the system and enable comparisons of wait times across the system.
 
+To collect the Performance metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.
```
Comment (alaudazzi) [inline packages/oracle/_dev/build/docs/README.md:150]:
```suggestion
To collect the Performance metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```

**Review by alaudazzi — COMMENTED** [review]
Left a few editing suggestions.
Regarding accessible terms, on top of the [WCAG 2.0 guidelines](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#guidelines) it is useful to refer to the Accessibility guidelines developed by the Elastic doc team.

**Review by alaudazzi — APPROVED** [review]
LGTM

**Review by ishleenk17 — APPROVED** [review]
Considering the permissions required for each of the metrics have been verified, the documentation looks fine.

**Pull request merged**